microcosm_postgres/store.py

```diff
@@ -200,6 +204,12 @@ def _filter(self, query, **kwargs):
         :param limit: pagination limit, if any
 
         """
+        for key, value in kwargs.items():
+            field = self.auto_filters.get(key)
+            if not field:
+                continue
+            query = query.filter(field == value)
```

Comment:
I think our default behavior is usually:
```
if value is not None:
    query = ...
```
(and might be worth adding a `Company.search(name=None)` test below for clarity either way?)